specs/card.proto

```diff
@@ -42,7 +42,7 @@ message StartDayExtension {
 message CardExtensions {
   optional RegionExtension extension_region = 1;
   optional BirthdayExtension extension_birthday = 2;
-  optional NuernbergPassNumberExtension extension_nuernberg_pass_number = 3;
```

Comment:
adding here an enum would be nice whether its PassID or PassNr and set the passId for now

Reply:
I think it would make sense to make PassNr the default (i.e. the one at position 0), as we cannot set it for older cards.